src/cdk/drag-drop/drag-ref.ts

```diff
@@ -66,7 +66,6 @@ interface DragHelperTemplate<T = any> {
 
 /**
  * Reference to a draggable item. Used to manipulate or dispose of the item.
- * @docs-private
```

Comment:
We should do the same for the `DropListRef`.

Reply:
Want me to include this in this PR or separate?

Reply:
Doing it in this PR is fine.

Reply:
Done 👍